### ISSUE TITLE
npm install problems

### ISSUE THREAD

**Issue by owang** [jamesmorgan/devicecast#1]
When I try to run ` npm install`, I get the following errors:

```
../src/bindings.cc:57:25: error: member reference type 'v8::Persistent<v8::FunctionTemplate>' is not a pointer; did you mean to use '.'?
                    s_ct->GetFunction());
                    ~~~~^~
                        .
../src/bindings.cc:57:27: error: no member named 'GetFunction' in 'v8::Persistent<v8::FunctionTemplate, v8::NonCopyablePersistentTraits<v8::FunctionTemplate> >'
                    s_ct->GetFunction());
                    ~~~~  ^
../src/bindings.cc:56:33: error: no member named 'NewSymbol' in 'v8::String'
        target->Set(v8::String::NewSymbol("input"),
                    ~~~~~~~~~~~~^
```

**Comment by jamesmorgan** [comment]
I wonder if this is related to an issue with one of the dependencies. I created and tested this using a specific version of node (see: https://github.com/jamesmorgan/devicecast#development) due to this same issue. 

I believe the c bindings inside one of the libs dont work with higher versions of node. 

I am unsure if this has been fixed, many be try diagnose which library and bump to the latest version, is may have been fixed. 

Closing for now until we know more.
